.travis.yml

```diff
@@ -2,7 +2,6 @@ language: ruby
 cache: bundler
 sudo: false
 rvm:
-  - 1.9.3-p551
```

Comment:
Did you mean to remove this? I think some companies still run this version (I know ...)

Reply:
just replied below in the comments. sounds good to you?